seatunnel-examples/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>seatunnel</artifactId>
+        <groupId>org.apache.seatunnel</groupId>
+        <version>2.0.5-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>seatunnel-examples</artifactId>
+    <packaging>pom</packaging>
+    <modules>
+        <module>seatunnel-flink-examples</module>
+    </modules>
+
+
+
```

Comment:
Please check it as a whole and remove useless spaces, thanks

Reply:
> Please check it as a whole and remove useless spaces, thanks

fix